TestApp.Wpf/BasicTestsWindow.xaml.cs

```diff
@@ -82,5 +83,20 @@ private void BtnTestStream_OnClick(object sender, RoutedEventArgs e)
                 }
             }
         }
+
+        private async void BtnTestImageBrush_OnClick(object sender, RoutedEventArgs e)
```

Comment:
See this comment: https://github.com/XamlAnimatedGif/XamlAnimatedGif/pull/52/files#r52102478